webots_ros2_importer/package.xml

```diff
@@ -22,9 +22,6 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pycodestyle</test_depend>
-  <test_depend>python3-pil</test_depend>
-  <test_depend>python-numpy</test_depend>
-  <test_depend>python-imaging</test_depend>
```

Comment:
Why deleting this?

Reply:
From the commit message it seems that those broke python3 compatibility?

@yanick-douven any insights?